src/commands/deployWorkspaceProject/getDefaultValues/getDefaultContextValues.ts

```diff
@@ -32,18 +32,16 @@ export async function getDefaultContextValues(
     const settings: DeployWorkspaceProjectSettingsV1 = await dwpSettingUtilsV1.getDeployWorkspaceProjectSettings(rootFolder);
     dwpSettingUtilsV1.setDeployWorkspaceProjectSettingsTelemetry(context, settings);
 
-    if (!context.ignoreExistingDeploySettings) {
-        // Logic to display local workspace settings related outputs
-        if (triggerSettingsOverride(settings, item)) {
-            // Tree item & settings conflict
-            context.telemetry.properties.settingsOverride = 'triggered';
-            await displaySettingsOverrideWarning(context, item as ContainerAppItem | ManagedEnvironmentItem);
-            context.telemetry.properties.settingsOverride = 'accepted';
-        } else {
-            // No settings conflict
-            context.telemetry.properties.settingsOverride = 'none';
-            dwpSettingUtilsV1.displayDeployWorkspaceProjectSettingsOutput(settings);
-        }
+    // Logic to display local workspace settings related outputs
+    if (triggerSettingsOverride(settings, item)) {
```

Comment:
Do we still need this logic? I'm assuming that we won't ever overwrite workspace deployment settings anymore.

Reply:
Not really, but I'm keeping it here because it's necessary to make V1 work which is what is still hooked up until we finish V2 and swap it over :)